_posts/2020-10-30-snowflake-dataOps.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: DataOps with AWS and Snowflake
+date: 2020-10-27
+tags: devops, dataops, aws, snowflake, ci, cd, flyway
+author: Zainab Maleki
+image: img/blog/dataops/dataOps.jpg
+---
+
+<center><img src="/img/blog/dataops/dataOps.jpg" /></center><br/>
+
+# What is DataOps?
+
+DevOps practices have been known, utilised and perfected over the past few years to allow speed and agility in the development life cycles. While all the advancements have happened in development lands, data domains have not kept pace. DataOps is a concept that allows applying DevOps practices to data life cycles to not only achieve speed and agility but also to deliver trusted and high-quality data.<br/>
```

Comment:
the negative 'to not only achieve is' difficult to comprehend while skimming, and there are a few grammar problems as well.

`DevOps practices have been discovered, utilised and perfected over the past few years to allow speed and agility in software engineering. Despite these advancements in the software engineering discipline, they have not been widely adopted in the data engineering domain. DataOps is a concept that applies DevOps practices to data life cycles to achieve speed and agility whilst delivering trusted and high-quality data.`